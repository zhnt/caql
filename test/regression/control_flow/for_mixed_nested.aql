// Mixed nested loops (numeric + range)
for i = 1, 3, 1 {
    for j in range(0, 2, 1) {
        print(i * 100 + j)
    }
}
