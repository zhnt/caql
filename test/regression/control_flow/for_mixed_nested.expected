100
101
200
201

